pkg/router/router.go

```diff
@@ -157,6 +162,10 @@ func routerContainer(cr *apiv1alpha1.PerconaServerMySQL) corev1.Container {
 			},
 		},
 		Ports: []corev1.ContainerPort{
+			{
```

Comment:
@egegunes please fix conflicts and add comment about this port. We need to be sure that nobody changes port order

Reply:
Maybe we don't need to ensure port order, we can set this annotation which tells AWS LB which port to use for health checks: `service.beta.kubernetes.io/aws-load-balancer-healthcheck-port`

Reply:
@egegunes  as I know aws uses targetPort to monitor LB I do not know how to get it before service creation